src/types.ts

```diff
@@ -60,6 +60,16 @@ export interface QuizMetadata {
   topic?: string;
 }
 
+export interface QuestionBucket {
+  bucketStartIndex: number,
+  bucketEndIndex: number,
```

Comment:
bucket is already in the type name no?
```suggestion
  start: number,
  end: number,
```